### ISSUE TITLE
[GUI] Can we have a green tick ✅ when checking an address on HW?

### ISSUE THREAD

**Issue by kloaec** [wizardsardine/liana#968]
We currently don't display anything on the GUI after the address check is performed on the HW. Could we add a green tick ✅ when it's done?
Maybe it's not technically possible, but it feel weird we do it for registration and signing, but not address verif.

![image](https://github.com/wizardsardine/liana/assets/15522963/5210bb4d-0d96-499e-a07e-390c067e2ed3)

**Comment by pythcoiner** [comment]
i think so, to me its just a way to send adress derivation path in order to launch verification process

**Comment by kloaec** [comment]
Good point Ed. But then the interface kinda looks weird for this purpose,
we are reusing the Signing/registering pop-up style but this time nothing
happens when you approve. Inconsistency imho.
